PFERD/crawl/ilias/ilias_web_crawler.py

```diff
@@ -156,7 +187,14 @@ def __init__(
 
         self._auth = auth
         self._base_url = section.base_url()
-        self._client_id = section.client_id()
+        self._tfa_auth = section.tfa_auth(authenticators)
+
+        self._login_type = section.login()
+        if isinstance(self._login_type, LocalLoginType):
+            self._client_id = self._login_type.client_id
+        else:
+            self._shibboleth_login = ShibbolethLogin(
+                self._base_url, self._auth, self._tfa_auth)
```

Comment:
```suggestion
            self._shibboleth_login = ShibbolethLogin(self._base_url, self._auth, self._tfa_auth)
```
PFERD uses a 110 max line length and I think that flows nicer in one line.